pkg/scheduler/framework/plugins/README.md

```diff
@@ -58,7 +62,7 @@ func SetDefaults_FooPluginArgs(obj *v1beta1.FooPluginArgs) {
 ```
 
 Remember to write tests in `defaults_test.go` (the test may not pass until you
```

Comment:
Would it be worth to introduce code generation and then explain required tests?

Reply:
Makes sense. I've moved things around a bit + added headers to make things more structured.